File: 1287-element-appearing-more-than-25-in-sorted-array/1287-element-appearing-more-than-25-in-sorted-array.java
class Solution {
    public int findSpecialInteger(int[] arr) {
       int arrLength = arr.length;
       int divide = (int)Math.floor(arrLength/4);

       for(int i = 0; i < (arrLength - divide); i++){
           if(arr[i] == arr[i + divide]){
             return arr[i];
           } 
       } 
    return -1; 
    } 
};

File: 1287-element-appearing-more-than-25-in-sorted-array/1287-element-appearing-more-than-25-in-sorted-array.js
/**
 * @param {number[]} arr
 * @return {number}
 */
function findSpecialInteger(arr) {
   const arrLength = arr.length;
   const divide = Math.floor(arrLength / 4);
   for(let i = 0; i < (arrLength - divide); i++){
       if(arr[i] === arr[i + divide]) return arr[i]
   }
};

 